.github/workflows/hugo-versioned-docs.yaml

```diff
@@ -1,19 +0,0 @@
-name: Hugo Versioned Docs
```

Comment:
Deleting this, because we do not use this anywhere and it is triggered manually.